static-assets/components/cstudio-admin/mods/content-type-propsheet/richText.js

```diff
@@ -161,6 +161,7 @@ YAHOO.extend(
       tinymce.init({
         target: inputEl,
         promotion: false,
+        branding: false,
```

Comment:
There are two props to hide those kind of content:
- `promotion`: to hide a premium version related promotion button - https://www.tiny.cloud/docs/tinymce/latest/editor-premium-upgrade-promotion/
- `branding`: to hide their `"Build with tinyMCE"` message - https://www.tiny.cloud/docs/tinymce/latest/statusbar-configuration-options/#branding